message-index/messages/S-4804/does-not-match/before/package.yaml

```
@@ -0,0 +1,15 @@
+name:                blah
+
+dependencies:
+- base >= 4.7 && < 5
+- say >= 0.2.0.1
```

What about picking a ludicrously large version number here, just so the example remains valid after the next release?
```suggestion
- say >= 100.2.0.1
```